scripts/prevent-idle-timeout.sh

```diff
@@ -8,27 +8,30 @@
 # SPDX-License-Identifier: EPL-2.0
 #
 
-# Ensure that main environment variable are configured
-if [ -z "${CHE_API_INTERNAL}" ]; then
-    echo "CHE_API_INTERNAL is not configured in environment."
+CA_CERT="/tmp/che/secret/ca.crt"
+
+
+if [ -z "${CHE_API_INTERNAL}" ] || [ -z "${CHE_WORKSPACE_ID}" ]; then
+    echo "CHE_API_INTERNAL or CHE_WORKSPACE_ID is not configured in environment."
     exit 1
 fi
-if [ -z "${CHE_MACHINE_TOKEN}" ]; then
-    echo "CHE_MACHINE_TOKEN is not configured in environment."
-    exit 1
+
+CURL_OPTS=
+
+if [ -e $CA_CERT ]; then
+	CURL_OPTS="$CURL_OPTS"' --cacert /tmp/che/secret/ca.crt'
 fi
-if [ -z "${CHE_WORKSPACE_ID}" ]; then
-    echo "CHE_WORKSPACE_ID is not configured in environment."
-    exit 1
+
+if [ -n "$CHE_MACHINE_TOKEN" ]; then
+	CURL_OPTS="$CURL_OPTS"' -H "Authorization: Bearer '"$CHE_MACHINE_TOKEN"'"'
 fi
 
-# Todo(vzhukovs): check for client certificates that might be included into request in case if script runs on minikube with self-signed certificates
+REQUEST_URL="$CHE_API_INTERNAL"'/activity/'"$CHE_WORKSPACE_ID"
 
-# Start making requests to avoid idling, make a request to activity tracker each minute
 while true; do
-    curl -X PUT \
-    -H "Authorization: Bearer ${CHE_MACHINE_TOKEN}" \
-    "${CHE_API_INTERNAL}/activity/${CHE_WORKSPACE_ID}"
+	echo "$(date) - Perform request: 'curl -X PUT $CURL_OPTS $REQUEST_URL'."
+	eval curl -X PUT "${CURL_OPTS}" "${REQUEST_URL}"
+	echo "$(date) - Sleep for 1 minute."
 
-    sleep 1m
+    sleep 60
```

Comment:
seems that now you like tab characters instead of spaces

Reply:
Yeah, didn't noticed that. Just used a different editor. Updated

Reply:
fwiw you can suppress whitespace changes when reviewing a PR with `?w=1` suffix, eg., https://github.com/che-incubator/jetbrains-editor-images/pull/15/files?w=1

Reply:
@nickboldt good to know, thanks, but agree, tab are evil

Reply:
Re: tabs v. spaces:
https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f

If we're going to change our language to be more inclusive (master/slave/blacklist etc.) we should also not fear being more inclusive with tabs. Just sayin'... they're not evil.